src/plugins/data/public/index_patterns/index_patterns/index.ts

```diff
@@ -17,9 +17,5 @@
  * under the License.
  */
 
-export * from './flatten_hit';
-export * from './format_hit';
-export * from './index_pattern';
-export * from './index_patterns';
-export * from './index_patterns_api_client';
-export * from './types';
+export * from '../../../common/index_patterns/index_patterns';
```

Comment:
Same question, should we re-export from here?

Reply:
Addressed other instances but this one is giving me trouble.